shell/platform/embedder/embedder.cc

```diff
@@ -731,6 +750,10 @@ FlutterEngineResult FlutterEngineSendPointerEvent(
     pointer_data.change = ToPointerDataChange(
         SAFE_ACCESS(current, phase, FlutterPointerPhase::kCancel));
     pointer_data.kind = flutter::PointerData::DeviceKind::kMouse;
+    // TODO(tong): Change 0 to a SAFE_ACCESS to current.buttons once this
+    // field is added.
+    // See https://github.com/flutter/flutter/issues/32052#issuecomment-489278965
+    pointer_data.buttons = ToPointerDataButtons(0, pointer_data.change);
```

Comment:
Since the button field will be at the end, let's put this line at the end as well; having them in a different order here seems like it introduces potential confusion.